### PR TITLE
Kindigo

### PR DESCRIPTION
**Pull request by prudhvid**
Integrated with g500 simulator

### CODE REVIEW

File: sensor_stack/seabotix/src/control.py [hunk 1]
```diff
@@ -37,34 +40,61 @@ def imuCB(dataIn):
...
+	fabserror=abs(errorP)
+	sign=1;
+	if(errorP<0):
+		sign=-1;
+	if(fabserror>180):
+		errorP=-1*(sign*(360-fabserror));
+	
+
+
+	print "Error P:", errorP, "; Yaw: ", yaw
```
Comment (icyflame) [inline sensor_stack/seabotix/src/control.py:52]: Please use `ros` logging functions. Refer https://github.com/auviitkgp/kraken_3.0/issues/70#issuecomment-143484124

File: sensor_stack/seabotix/src/control.py [hunk 2]
```diff
@@ -37,34 +40,61 @@ def imuCB(dataIn):
 
 	prevError = errorP
 	errorP = goal - yaw
-	print errorP
+	fabserror=abs(errorP)
+	sign=1;
+	if(errorP<0):
+		sign=-1;
```
Comment (icyflame) [inline sensor_stack/seabotix/src/control.py:46]: @prudhvid @kalyan-kumar Are semi-colons really necessary? :stuck_out_tongue:  It is Python, after all.

**Comment by prudhvid** [comment]
@icyflame 
Those files aren't meant to be there. Thanks

**Comment by kalyan-kumar** [comment]
I was still struggling with getting the code :/

**Comment by prudhvid** [comment]
check the new PR in main repo and test that

On Tue, Sep 29, 2015 at 1:22 AM, Kalyan Kumar notifications@github.com
wrote:

> I was still struggling with getting the code :/
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kalyan-kumar/kraken_3.0/pull/1#issuecomment-143856741
> .

## 

_Prudhvi Dharmana_

_B.Tech 4th year_
_Dept. of Computer Science & Engineering_
_IIT Kharagpur._

**Comment by kalyan-kumar** [comment]
Yaa. Doing it.

**Comment by icyflame** [comment]
@kalyan-kumar @prudhvid has opened a new PR: https://github.com/auviitkgp/kraken_3.0/pull/71

**Pull request closed without merging**